\documentclass{article}
\begin{document}
SNV/Indel calls were further filtered through a Bayesian classifier to select only those sites that were classified as somatic. Briefly, this classifier works by considering the likelihood of the data being generated by 7 different binomial models. Each model's log-likelihood function takes the form:
\[ 

\log \mathcal{L} (N_r, N_v, T_r, T_v) = 
N_r \log (1 - \theta_N) + N_v \log \theta_N + T_r \log (1 - \theta_T) + T_v \log \theta_T

\]

where $N_r$ is the number of reads supporting the reference allele in the normal sample,
$N_v$ is the number of reads supporting the variant allele in the normal sample,
$T_r$ is the number of reads supporting the reference allele in the tumor sample,
$T_v$ is the number of reads supporting the variant allele in the tumor sample.

Each of the different models defines $\theta_N$ and $\theta_T$ as defined below:
\[

(\theta_N, \theta_T) = 
 \begin{cases}
   \epsilon, \epsilon & \quad \text{ for the Reference model }\\
   0.5, 0.5 & \quad \text{ for the Germline heterozygote model }\\
   1 - \epsilon, 1 - \epsilon & \quad \text{ for the Germline homozygote model }\\
   \epsilon + \frac{f_t c_n}{p_t}, f_t & \quad \text{ for the Somatic model }\\
   0.5, 1 - \epsilon & \quad \text{ for the LOH variant model }\\
   0.5, \epsilon & \quad \text{ for the LOH reference model }\\
   f_s, f_s & \quad \text{ for the Noise model }\\
 \end{cases}
\]

where \epsilon is a constant error rate of 1\%, 
$f_t$ is the observed variant allele frequency in the tumor,
$f_s$ is the observed variant allele frequency at the site (across both samples),
$c_n$ is the contamination rate of the normal sample in the tumor,
and $p_t$ is the purity of the tumor sample.

Sites pass the filter if the Somatic model has the maximum likelihood amongst all models and the log-likelihood ratio of the Somatic model to the next most likely model is greater than 3.
\end{document}        
